packages/teleterm/src/services/pty/ptyService.ts

```diff
@@ -66,3 +67,19 @@ function buildOptions(settings: RuntimeSettings, cmd: PtyCommand): PtyOptions {
 
   throw Error(`Unknown pty command type: ${cmd.kind}`);
 }
+
+function getDefaultShell(): string {
```

Comment:
lets move this method to the [runtimeSettings.ts](https://github.com/gravitational/webapps/blob/gzdunek/use-default-terminal-shell-and-add-context-menu/packages/teleterm/src/mainProcess/runtimeSettings.ts#L10)

Reply:
Done